dace/transformation/dataflow/streaming_memory.py

```diff
@@ -263,13 +359,100 @@ def apply(self, sdfg: SDFG) -> nodes.AccessNode:
         streams = {}
         mpaths = {}
         for edge in edges:
-            name, newdesc = sdfg.add_stream(dnode.data,
-                                            desc.dtype,
-                                            buffer_size=self.buffer_size,
-                                            storage=self.storage,
-                                            transient=True,
-                                            find_new_name=True)
-            streams[edge] = name
+
+            if self.use_memory_buffering:
+
+                arrname = str(self.access(sdfg))
+
+                # Add gearbox
+                total_size = edge.data.volume
+                vector_size = int(self.memory_buffering_target_bytes /
+                                  desc.dtype.bytes)
+
+                if isinstance(sdfg.arrays[dnode.data].shape[-1], dace.symbol):
+                    warnings.warn(
```

Comment:
Maybe we should try to resolve the symbol instead and fail if this is not the case. @definelicht what do you think?